doc/rfc-move-sigverify-out-of-mask-rom.md

```diff
@@ -0,0 +1,80 @@
+# RFC: Move sigverify out of the mask ROM
+
+Author: James Wainwright
+Date: 2025-04-17
+Status: draft
+
+## Summary
+
+This RFC proposes moving the silicon creator signature verification code from the mask ROM to the immutable ROM extension, which would no longer be optional nor signed.
```

Comment:
In order to support multiple behaviors within the project we'll need to start coining names for behavior and introducing optionality that can be selected for both the HW and SW behavior. Has anyone started thinking about deployments of different behaviors?